pkg/kf/apps/push.go

```diff
@@ -101,7 +101,26 @@ func (p *pusher) Push(appName string, opts ...PushOption) error {
 	var hasDefaultRoutes bool
 	app.Spec.Routes, hasDefaultRoutes = setupRoutes(cfg, app.Name, app.Spec.Routes)
 
-	resultingApp, err := p.appsClient.Upsert(app.Namespace, app, mergeApps(cfg, hasDefaultRoutes))
+	// Scaling
+	if (cfg.MinScale != nil && cfg.MaxScale != nil) &&
+		(*cfg.MinScale == *cfg.MaxScale) {
+		// Exactly
+		app.Spec.Instances.Exactly = cfg.MinScale
```

Comment:
Let's pass along exactly as an option too or just construct the scaling object in push and pass the whole thing rather than encoding it in there and decoding it here.